apps/docs/src/app/core/component-docs/moment-datetime-adapter/examples/moment-datetime-formats-example.component.ts

```diff
@@ -0,0 +1,33 @@
+import { Component } from '@angular/core';
+import { DATE_TIME_FORMATS } from '@fundamental-ngx/core';
+import moment, { Moment } from 'moment';
+
+@Component({
+    selector: 'fd-moment-datetime-formats-example',
+    templateUrl: './moment-datetime-formats-example.component.html',
```

Comment:
The template isn't set inline and it's not shown on the separate tab in examples, when opening StackBlitz template file (`.html`) isn't loaded also.

Reply:
I think multiple datepickers utilizing different formats should be used in the example to better demonstrate how the formatter works, including a date range picker for the delimiter option.  So we will also want to display the HTML for that... this is handled in the `moment-datetime-adapter-docs.component.ts` file ExampleFiles[]